lib/vintage_net_wizard/backend/default.ex

```diff
@@ -88,6 +88,7 @@ defmodule VintageNetWizard.Backend.Default do
         {VintageNet, ["interface", "wlan0", "wifi", "access_points"], _, access_points, _},
         state
       ) do
+    access_points = Enum.map(access_points, &Map.from_struct/1)
```

Comment:
Was the `__struct__` field causing issues?

Reply:
When calling `Jason.ecode` it expects the struct to have the `Jason.Encoder` protocol on it and I assumed that doing that would be more expected then implementing the protocol.

Reply:
Makes sense.